package/MDAnalysis/core/groups.py

```diff
@@ -3010,7 +3014,18 @@ def select_atoms(self, sel, *othersel, periodic=True, rtol=1e-05,
             smarts *SMARTS-query*
                 select atoms using Daylight's SMARTS queries, e.g. ``smarts
                 [#7;R]`` to find nitrogen atoms in rings. Requires RDKit.
-                All matches (max 1000) are combined as a unique match
+                All matches are combined as a single unique match. The `smarts`
+                selection accepts two sets of key word arguments from
+                `select_atoms()`: the rdkit_kwargs are passed internally to
```

Comment:
probably will want double ticks around `rdkit_kwargs` and `smart_kwargs`

As you mentioned this is not being updated in the main docs, I _think_ you need to directly edit: doc/sphinx/source/documentation_pages/selections.rst

(not sure why there are two versions of this, probably historical things)

Reply:
Thanks, I was wondering where I needed to make those changes!